genisys/modules/kernelparameter.py

```diff
@@ -0,0 +1,27 @@
+from typing import Self
+from pathlib import Path
+
+from genisys.modules import base
+from typing import Self
```

Comment:
You have this import twice. Also for compatibility with earlier Python versions, I've discovered we should be using `typing_extensions` which you'll have to install separately. I'll make that change in my own module's as well.

Reply:
Removed extra imports and added typing_extensions.